lua/neogit/lib/git/remote.lua

```diff
@@ -24,9 +24,10 @@ local function cleanup_push_variables(remote, new_name)
 end
 
 function M.add(name, url, args)
+  print("Adding remote", name, url, vim.inspect(args))
```

Comment:
I don't think this is strictly needed (no need to make it too noisy) but if you think so, lets do a notification.

Reply:
Thought I cleared those out. Couldn't reliably get the logging for the tests to show up in the right order when all were run. Will need to expand the testing `init.lua` to support running just 1 test

Reply:
_agreed_, thats a pain point for me too.

Reply:
`NEOGIT_LOG_CONSOLE=sync` works somewhat better, as it uses `nvim_writeln` most of the time rather than a `vim.schedule(write)`, but may defer a write to the next frame if in a fast event.

The other issue is a way to run a single test easier.

`:PlenaryBustedFile %` does not work for me and ends up breaking my editor by making the test output lines interleave with the Neovim ui itself, causing a "staggering" for each printed terminal line. Don't know if you've had the same experience.

The `recipes.json` is a custom plugin I am using and is a crude way to solve it, though does not work with the new `tests/init.lua` which runs all tests anyway.

I think I'll wait for treaty breaker's minimal init and see where that goes, and then add the testing shim ontop of that. Having the same `init.lua` during testing *and* issue reproduction is golden.